documentation/library_guide/dynamic_selection_api/backends.rst

```diff
@@ -0,0 +1,279 @@
+Backends
+########
+
+The dynamic selection API is an experimental feature in the |onedpl_long|
+(|onedpl_short|) that selects an *execution resource* based on a chosen
+*selection policy*. A *backend* is responsible for managing a set of resources
+and handling the submission of work to those resources.
+
+Backends are typically not directly visible to application developers - they work
+behind the scenes to enable policies to function.
+
+Role of Backends
+----------------
+
+A backend serves several key purposes:
+
+1. **Resource Management** - Stores and provides access to a set of resources
+2. **Work Submission** - Handles the execution of user functions on selected resources
+3. **Instrumentation** - Optionally reports execution information (timing, completion) to policies
+4. **Synchronization** - Provides mechanisms to wait for submitted work to complete
+
+The dynamic selection API separates the *selection logic* (in policies) from the
+*resource management and submission mechanics* (in backends), allowing policies
+to work with different resource types.
+
+Default SYCL Backend
+--------------------
+
+By default, policies use the SYCL backend which manages ``sycl::queue`` resources.
+The SYCL backend provides:
+
+- Default initialization from available SYCL devices
+- Event-based submission and waiting
+- Optional profiling and timing instrumentation
+
+When you construct a policy without specifying resources, it uses the SYCL
+backend's default initialization:
+
+.. code:: cpp
+
+  #include <oneapi/dpl/dynamic_selection>
+
+  namespace ex = oneapi::dpl::experimental;
+
+  // Uses SYCL backend with default device initialization
+  ex::round_robin_policy<> policy;
+
+  // Explicitly specifying SYCL queues
+  std::vector<sycl::queue> my_queues = { /* ... */ };
+  ex::round_robin_policy<sycl::queue> policy2{my_queues};
+
+Backend Template Parameters
+---------------------------
+
+Policies accept a backend template parameter to control the resource type:
+
+.. code:: cpp
+
+  template<typename ResourceType = sycl::queue,
+           typename ResourceAdapter = oneapi::dpl::identity,
+           typename Backend = default_backend<ResourceType, ResourceAdapter>>
+  class round_robin_policy { /* ... */ };
+
+- **ResourceType** - The type of resources managed (default: ``sycl::queue``)
+- **ResourceAdapter** - Adapter to transform resources (default: ``oneapi::dpl::identity``)
+- **Backend** - The backend implementation (default: ``default_backend``)
+
+Most users only need to specify ``ResourceType`` if using non-SYCL resources.
+
+Resource Adapters
+-----------------
```

Comment:
This section repeats the information on the "Custom Backends" page almost word-to-word. I would instead move it also to the page about policies (perhaps the customization section there) and tried to focus on different aspects and/or vary the level of details instead of full repetition.

Reply:
I've removed the custom_backends.rst page and kept what seems useful on this page which relates to backends.